### ISSUE TITLE
License scan failed for npm:alg-notifications:1.0.1

### ISSUE THREAD

**Issue by fridex** [fabric8-analytics/fabric8-analytics-worker#321]
```
{"dispatcher_id": "76b5f001-3ed1-408f-bb72-ae7df7cecfb3", "task_id": "daba5fe6-b681-45e4-929f-d148d6714c56", "event": "TASK_START", "parent": {}, "queue": "prod_ingestion_LicenseCheckTask_v0", "node_args": {"document_id": 3169279, "version": "1.0.1", "name": "alg-notifications", "_audit": {"version": "v1", "started_at": "2017-09-11T16:04:43.797275", "ended_at": "2017-09-11T16:04:49.296206"}, "_release": "npm:alg-notifications:1.0.1", "ecosystem": "npm"}, "flow_name": "bayesianAnalysisFlow", "task_name": "source_licenses"}
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib64/python3.4/logging/__init__.py", line 978, in emit
    msg = self.format(record)
  File "/usr/lib64/python3.4/logging/__init__.py", line 828, in format
    return fmt.format(record)
  File "/usr/lib64/python3.4/logging/__init__.py", line 573, in format
    record.exc_text = self.formatException(record.exc_info)
  File "/usr/lib64/python3.4/logging/__init__.py", line 523, in formatException
    traceback.print_exception(ei[0], ei[1], tb, None, sio)
  File "/usr/lib64/python3.4/traceback.py", line 169, in print_exception
    for line in _format_exception_iter(etype, value, tb, limit, chain):
  File "/usr/lib64/python3.4/traceback.py", line 146, in _format_exception_iter
    for value, tb in values:
  File "/usr/lib64/python3.4/traceback.py", line 125, in _iter_chain
    context = exc.__context__
AttributeError: 'NoneType' object has no attribute '__context__'
Call stack:
  File "/usr/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python3.4/site-packages/celery/__main__.py", line 14, in main
    _main()
  File "/usr/lib/python3.4/site-packages/celery/bin/celery.py", line 326, in main
    cmd.execute_from_commandline(argv)
  File "/usr/lib/python3.4/site-packages/celery/bin/celery.py", line 488, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/usr/lib/python3.4/site-packages/celery/bin/base.py", line 281, in execute_from_commandline
    return self.handle_argv(self.prog_name, argv[1:])
  File "/usr/lib/python3.4/site-packages/celery/bin/celery.py", line 480, in handle_argv
    return self.execute(command, argv)
  File "/usr/lib/python3.4/site-packages/celery/bin/celery.py", line 412, in execute
    ).run_from_argv(self.prog_name, argv[1:], command=argv[0])
  File "/usr/lib/python3.4/site-packages/celery/bin/worker.py", line 221, in run_from_argv
    return self(*args, **options)
  File "/usr/lib/python3.4/site-packages/celery/bin/base.py", line 244, in __call__
    ret = self.run(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/celery/bin/worker.py", line 256, in run
    worker.start()
  File "/usr/lib/python3.4/site-packages/celery/worker/worker.py", line 203, in start
    self.blueprint.start(self)
  File "/usr/lib/python3.4/site-packages/celery/bootsteps.py", line 119, in start
    step.start(parent)
  File "/usr/lib/python3.4/site-packages/celery/bootsteps.py", line 370, in start
    return self.obj.start()
  File "/usr/lib/python3.4/site-packages/celery/worker/consumer/consumer.py", line 318, in start
    blueprint.start(self)
  File "/usr/lib/python3.4/site-packages/celery/bootsteps.py", line 119, in start
    step.start(parent)
  File "/usr/lib/python3.4/site-packages/celery/worker/consumer/consumer.py", line 594, in start
    c.loop(*c.loop_args())
  File "/usr/lib/python3.4/site-packages/celery/worker/loops.py", line 88, in asynloop
    next(loop)
  File "/usr/lib/python3.4/site-packages/kombu/async/hub.py", line 345, in create_loop
    cb(*cbargs)
  File "/usr/lib/python3.4/site-packages/kombu/async/http/curl.py", line 115, in on_readable
    return self._on_event(fd, _pycurl.CSELECT_IN)
  File "/usr/lib/python3.4/site-packages/kombu/async/http/curl.py", line 128, in _on_event
    self._process_pending_requests()
  File "/usr/lib/python3.4/site-packages/kombu/async/http/curl.py", line 134, in _process_pending_requests
    self._process(curl)
  File "/usr/lib/python3.4/site-packages/kombu/async/http/curl.py", line 182, in _process
    buffer=buffer, effective_url=effective_url, error=error,
  File "/usr/lib/python3.4/site-packages/vine/promises.py", line 146, in __call__
    svpending(*ca, **ck)
  File "/usr/lib/python3.4/site-packages/vine/promises.py", line 136, in __call__
    retval = fun(*final_args, **final_kwargs)
  File "/usr/lib/python3.4/site-packages/vine/funtools.py", line 100, in _transback
    return callback(ret)
  File "/usr/lib/python3.4/site-packages/vine/promises.py", line 136, in __call__
    retval = fun(*final_args, **final_kwargs)
  File "/usr/lib/python3.4/site-packages/vine/funtools.py", line 100, in _transback
    return callback(ret)
  File "/usr/lib/python3.4/site-packages/vine/promises.py", line 136, in __call__
    retval = fun(*final_args, **final_kwargs)
  File "/usr/lib/python3.4/site-packages/vine/funtools.py", line 96, in _transback
    ret = filter_(*args + (ret,), **kwargs)
  File "/usr/lib/python3.4/site-packages/kombu/transport/SQS.py", line 324, in _on_messages_ready
    callbacks[qname](message)
  File "/usr/lib/python3.4/site-packages/kombu/transport/virtual/base.py", line 633, in _callback
    return callback(message)
  File "/usr/lib/python3.4/site-packages/kombu/messaging.py", line 623, in _receive_callback
    return on_m(message) if on_m else self.receive(decoded, message)
  File "/usr/lib/python3.4/site-packages/celery/worker/consumer/consumer.py", line 568, in on_task_received
    callbacks,
  File "/usr/lib/python3.4/site-packages/celery/worker/strategy.py", line 145, in task_message_handler
    handle(req)
  File "/usr/lib/python3.4/site-packages/celery/worker/worker.py", line 226, in _process_task
    req.execute_using_pool(self.pool)
  File "/usr/lib/python3.4/site-packages/celery/worker/request.py", line 531, in execute_using_pool
    correlation_id=task_id,
  File "/usr/lib/python3.4/site-packages/celery/concurrency/base.py", line 155, in apply_async
    **options)
  File "/usr/lib/python3.4/site-packages/celery/concurrency/base.py", line 31, in apply_target
    ret = target(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 530, in _fast_trace_task
    uuid, args, kwargs, request,
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 367, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 622, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 115, in run
    result = task.run(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/base.py", line 38, in run
    result = self.execute(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/license.py", line 113, in execute
    self.log.exception("License scan failed")
Message: 'License scan failed'
Arguments: ()
```

**Comment by fridex** [comment]
CC @jpopelka

**Comment by fridex** [comment]
I can see the same for npm:ep-tarantino:2.1.0.

**Comment by jpopelka** [comment]
I couldn't reproduce the problems locally, but the error reporting should be fixed with #323
